python/hsfs/feature_group.py

```diff
@@ -323,6 +382,12 @@ def update_features(self, features: Union[feature.Feature, List[feature.Feature]
     def update_feature_description(self, feature_name: str, description: str):
         """Update the description of a single feature in this feature group.
 
+        !!! example
+            ```python
+            updated_weather_fg = weather_fg.update_feature_description(feature_name="min_temp",
```

Comment:
again remind users what are `updated_weather_fg` and `weather_fg`? just show how did you fetch them